backend/src/index.ts

```diff
@@ -193,19 +204,41 @@ app.post("/login", async (req: TypedRequest<LoginBody>, res: Response) => {
 });
 // curl -H 'Content-Type: application/json' -d '{ "email": "ben", "color": "pink"}' -X POST http://localhost:3000/subscribe
 
-app.get('/startGame', async (req: TypedRequestQuery<{roundCount: number}>, res) => {
+app.get('/startGame', async (req: TypedRequestQuery<{roundCount: number, gameMode: Gamemode}>, res) => {
 
-    const getDoc = await getDocs(collection(db, 'test_locations'));
-    const allLocations = getDoc.docs.map(doc => doc.data);
+    const getDoc = await getDocs(collection(db, 'levels'));
+    const allLocations = getDoc.docs.map(doc => doc.data());
 
-    const n = req.query.roundCount;
-    // Shuffle array
+    const {roundCount, gameMode} = req.query;
     const shuffled = allLocations.sort(() => 0.5 - Math.random());
 
     // Get sub-array of first n elements after shuffled
-    let selected = shuffled.slice(0, n);
+    let selected = shuffled.slice(0, roundCount);
+    const levels : Level[] = [];
+    selected.forEach(location => 
+        levels.push({
+            photoLink: location.panorama,
```

Comment:
we'll have to make sure that we update this to reflect our actual image URLs in firebase storage. ill write a comment abt this